shared/xccdf/services/nfs.xml

```diff
@@ -86,17 +86,17 @@ this service should be disabled.
 
 </Group> <!-- end disabling_nfs -->
 
-<Group id="nfs_configuring_all_machines">
+<Group id="nfs_configuring_all_systems">
```

Comment:
Don't have a very strong opinion but a change seems OK given this is a group ID (vs rules, which are more likely to be selected/unselected) and the XCCDF rules within the nfs_configuring_all_machines group don't have ref tags (which would indicate someone along the way cared enough to map to OS SRG or NIST). They don't appear to be actively used in core profiles.